src/common/docker-compose/index.ts

```diff
@@ -203,7 +203,8 @@ export class DockerComposeUtils {
               compose.volumes[spec.key] = { external: true };
               volume = `${spec.key}:${service_volume}${spec.readonly ? ':ro' : ''}`;
             } else {
-              volume = service_volume;
+              volume = `${key}:${service_volume}`;
```

Comment:
is this right? if I don't specify a host_path I'd expect docker to create a volume
https://docs.docker.com/compose/compose-file/compose-file-v3/

```
volumes:
  # Just specify a path and let the Engine create a volume
  - /var/lib/mysql
  ````

Reply:
It would, but that volume would be anonymous. That wouldn't work if want volumes specified in a service to be persistent between redeploys

Reply:
ah that's what I was afraid of - annoying that anon volumes mean explicit and non-explicit. That makes no sense.